Esri/arcgis-runtime-samples-ios
# New: `Orbit camera around object`

## Description

This PR implements the `Orbit camera around object` sample. See details at `common-samples/issues/1188`

Branch URL: https://github.com/Esri/arcgis-runtime-samples-ios/tree/Ting/New-Orbit-vNext/arcgis-ios-sdk-samples/Scenes/Orbit%20camera%20around%20object

See a demo video below.

https://user-images.githubusercontent.com/9660181/127719794-b1811315-8d4e-45a3-90f9-721e77a9fa27.mp4

## Known issues

1. ~~If the animation for cockpit view is interrupted before finishing, there will be a timing bug. i.e. the camera may end up in an unexpected position. Since it doesn't matter too much, I didn't handle the cancelable job.~~ cancelable job is handled, and `animation finished` is checked.
2. ~~When the popover is presenting, touching on the scene view doesn't dismiss it. This is as designed, so that user can pan and zoom on the scene view to control the camera heading. To dismiss the popover, either tap on the nav bar, or on the button itself.~~ See discussion below.
3. The heading and pitch of the graphic is used instead of the symbol's. This is as designed, though I'm not sure why. (i.e. `planeGraphic.attributes["PITCH"]` is used instead of `planeSymbol.pitch`)

## How to test

Read through the README, and try all the UI controls to see if the rotation stuck at some point.
